studies/templates/studies/study_attachments.html

```diff
@@ -2,71 +2,55 @@
 {% load django_bootstrap5 %}
 {% load exp_extras %}
 {% load tz %}
+{% load static %}
+{% load bootstrap_icons %}
 {% block title %}
     Videos | {{ study.name }}
 {% endblock title %}
 {% block head %}
     {{ block.super }}
-    <script type="application/javascript">
-        $(document).ready(function () {
-            if ("{{ match }}" !== '') {
-                document.getElementById("search-attachments").value = "{{ match }}";
-            }
-        });
-    </script>
```

Comment:
@okaycj I know you mentioned removing this in other places. I think this can be safely removed here too, but not 100% sure.